code/game/objects/items/litmus.dm

```diff
@@ -0,0 +1,147 @@
+/obj/item/fermichem/pHbooklet
```

Comment:
we have stacks objects that do this, really should be using that.

Reply:
like?

Reply:
metal/bruise packs/etc.

Reply:
this is fine actually